docs/src/componentDocs/DrawerFooter/examples/ComplexDrawerFooter.tsx

```diff
@@ -39,9 +39,9 @@ export const ComplexDrawerFooter = (): JSX.Element => (
     <Box>
         <ComplexDrawerFooterExample />
         <CodeBlock code={codeSnippet} language="jsx" dataLine="9-30" />
-        <Box sx={{ display: 'flex', justifyContent: 'flex-end' }}>
-            <CopyToClipboard title={'Copy All'} copyText={codeSnippet} />
-            <FullCodeOnGithub sx={{ ml: 2 }} url="componentDocs/DrawerFooter/examples/ComplexDrawerFooterExample.tsx" />
-        </Box>
+        <CodeBlockActionButtonRow
+            copyText={codeSnippet}
+            url="componentDocs/DrawerFooter/examples/ComplexDrawerFooterExample.tsx"
+        />
```

Comment:
No need to do it now, but we may want to pull all these urls out into one config file and then read them here. This would allow us to modify them all in one place as a source of truth. Also makes for batch find and replace.

Reply:
[BLUI-3487](https://jira-prod.tcc.etn.com/browse/BLUI-3487)